specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -889,6 +889,58 @@
         }
       }
     },
+    "AcsCallParticipantAddedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallParticipantAdded event.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsCallParticipantEventData"
+        }
+      ],
+      "type": "object"
+    },
+    "AcsCallParticipantRemovedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallParticipantRemoved event.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsCallParticipantEventData"
+        }
+      ],
+      "type": "object",
+      "properties": {
+      }
```

Comment:
Why is this empty?

Reply:
you missing reason and removedBy? for participantRemovedEvent 
![image](https://github.com/user-attachments/assets/4e83cc68-8253-431b-bb47-f502f4689de8)